src/plugins/data_source_management/public/mocks.ts

```diff
@@ -0,0 +1,180 @@
+/*
```

Comment:
seems might having conflicts with this one: https://github.com/opensearch-project/OpenSearch-Dashboards/pull/2350/files

Reply:
I've aligned this file with the one in Mani's pr, so there shouldn't be any conflicts.